homeassistant/components/device_tracker/bmw_connected_drive.py

```diff
@@ -42,7 +42,8 @@ def update(self) -> None:
         attrs = {
             'trackr_id': dev_id,
```

Comment:
Is `trackr_id` used for something specific? As this field has the same value as `id` it seems a bit double.

Reply:
the attributes are removed in #12999